### DISCUSSION TITLE
Re-Implementing varied spawn loadouts

### DISCUSSION THREAD

**Discussion by Rehexr** [FakeFishGames/Barotrauma#9388] [Ideas and suggestions]
Seen this change just recently, so thought it makes sense to submit this. Simply having all loadouts be the same items is a bit bland, its much more prefferable if classes would still have small pieces of variety, such as a captian having a harmonica as part of a loadout and so on, while still following the nerf implemented. If anything, itd make sense to give sec the 3 different variations of headgear per loadout to give them some proper variety, avoiding overstocking them. This change is especially annoying with engineers and so, as theyre stuck without having options for wiring right away, locking away a lot of fun to be had in missions, where restocking isnt available unlike campaign.
Tldr Re-add varied, but still nerfed, start loadouts

**Comment by itchyOwl** [comment]
Thanks for the feedback.

The reason why we removed the variation between the loadouts was because they were not balanced (for example in terms of valuation). We also wanted to drastically remove the items that you start with in the campaign.

I agree that we should re-introduce some differences, and none of this makes any sense in single mission mode. But I'd rather try to define the loadouts so that they work in all the game modes.

To compensate the lack of items in the single mission mode, more items should now spawn in the tagged containers. I'll add some wires there too.
